### PR TITLE
Port to python 3

### PR DESCRIPTION
**Pull request by T-Sh**

### CODE REVIEW

File: util/parseRPMrepo.py [hunk 1]
```diff
@@ -1,80 +1,86 @@
...
+
 def parse_repomd(repo):
     filename = os.path.join(repo, "repodata", "repomd.xml")
     toret = {}
 
     tree = ElementTree()
     tree.parse(open(filename, 'r'))
 
-    for data in tree.findall(XMLREPONS+"data"):
+    for data in tree.findall(XMLREPONS + "data"):
```
Comment (TruePack) [inline util/parseRPMrepo.py:32]: Better way to string concatenation is format strings like f"{XMLREPONS}data} 
Same for other similar cases.

File: util/parseSources.py [hunk 2]
```diff
@@ -1,35 +1,37 @@
...
-    cur_param=""
+    pkg = {}
+    cur_param = ""
 
     for line in file:
         if line == "\n":
-            #print("----------------------------------------------------")
-            basedir=pkg['directory']
-            files=files_regex.findall(pkg['files'])
+            # print("----------------------------------------------------")
```
Comment (TruePack) [inline util/parseSources.py:29]: Debug?

File: util/parseSources.py [hunk 3]
```diff
@@ -40,5 +42,5 @@
             # We got a multiliner continuation
             pkg[cur_param] += line.lstrip()
         else:
-            print "IMMPOSSIBIRUUUU!!!!"
+            print("IMMPOSSIBIRUUUU!!!!")
```
Comment (TruePack) [inline util/parseSources.py:45]: Is it really necessary?

File: util/parseRPMrepo.py [hunk 4]
```diff
@@ -1,80 +1,86 @@
...
+    #    print(i.tag)
 
+    for package in tree.findall(XMLPKGNS + "package"):
+        location = package.find(XMLPKGNS + "location").get("href")
+        checksum = package.find(XMLPKGNS + "checksum").text
+        checksum_type = package.find(XMLPKGNS + "checksum").get("type")
+        sys.stdout.write(location + "\n")
+        sys.stderr.write(checksum_type + " " + checksum + " " + location + "\n")
 
+    # for location in tree.findall(XMLPKGNS+"package/"+XMLPKGNS+"location"):
```
Comment (TruePack) [inline util/parseRPMrepo.py:85]: Debug?

File: util/parseArchDB.py [hunk 5]
```diff
@@ -1,8 +1,7 @@
-#!/usr/bin/python
```
Comment (TruePack) [inline util/parseArchDB.py:1]: I think, you can refactor package names too with this [convention](https://www.python.org/dev/peps/pep-0008/#package-and-module-names).

File: util/parseSources.py (hunk 2 above)
Comment (T-Sh) [reply util/parseSources.py]: Removed all commented lines with debug info.

File: util/parseArchDB.py (hunk 5 above)
Comment (T-Sh) [reply util/parseArchDB.py:1]: Converted script's names to lowercase.

File: util/parseSources.py (hunk 3 above)
Comment (T-Sh) [reply util/parseSources.py]: Left only system exit, without additional info output.

**Comment by TruePack** [comment]
LGTM 
@redrampage can you merge mr?

**Comment by redrampage** [comment]
@TruePack unfortunately no. I no longer maintain this project and have no write permissions for this repo.

**Pull request merged**